recipes/eodie/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "EODIE" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url : https://gitlab.com/eetun-tiimi/EODIE/-/archive/v{{ version }}/EODIE-v{{ version }}.tar.gz
+  sha256: 09114493834794bb8836e6f1ccc5ff2bd2e8aed995a6cfbc8a68dffebbf32e73
+
+build:
+  skip: true  # [win]
```

Comment:
Why do you skip Windows? Its a Python package and it should work on Windows as well?

Reply:
There used to be some issues with gdal. So did not test it for long. Now it seems fine, just needed to update paths used in tests